### DISCUSSION TITLE
Using AutoHotkey to provide an alternative keyboard shortcut

### DISCUSSION THREAD

**Discussion by res2k** [res2k/HDRTray#29] [Show and tell]
Some people would like a keyboard shortcut to toggle HDR besides Win+Alt+B provided by the Xbox game bar.

At this time, I don't have plans to include this in HDRTray.
However, if you're already using AutoHotkey, @indigofairyx has snippet you can use as a starting point (from https://github.com/res2k/HDRTray/issues/17#issuecomment-2351495975):
```ahk
global hdrToggle := false
^#h::
    hdrToggle := !hdrToggle ; Toggle the state
    ; Check the state and run the corresponding command
    if (hdrToggle) {
        Run, "C:\Portable Programs\HDR Tray CMD\HDRCmd.exe" "on"
    } else {
        Run, "C:\Portable Programs\HDR Tray CMD\HDRCmd.exe" "off"
    }
Return
```

Disclaimer: I did not test this myself.

**Comment by indigofairyx** [comment]
Thanks for highlighting my snippet.

a couple quick notes on this code snippet, for those trying to use it...

- 1 its written in AutoHotkey v1
- 2 the `global hdrToggle := false` line must be placed in the top, auto execute, area of your script. if not it will errorout
- 3 the two lines that start with `Run, "C:\Portable Programs\HDR Tray CMD\HDRCmd.exe"` < -- these need to be updated with the path were you have HDRCmd.exe installed, also keep that path in "the quotes"
